src/OSCLedConfig.cpp

```diff
@@ -17,7 +17,7 @@
 using namespace std;
 
 // string constants used for config file parsing
-const string OSCLedConfig::KEY_DEFAULTS = "defaults";
+const string OSCLedConfig::KEY_DEFAULTS = "global";
```

Comment:
I decided "global" is a better keyword than "defaults" in the file. This is because some of the values (like "daemonize") are not defaults, they are a global setting.